docs/source/install-plone-restapi.rst

```diff
@@ -0,0 +1,77 @@
+Getting started with plone.restapi
+==================================
+If you haven't installed Plone locally, proceed to the next section, that is, installing Plone with plone.restapi. Make sure you have Python 2.7 and `virtualenv
+<https://virtualenv.pypa.io>`_ installed.
+If you have already set-up Plone locally, you can skip the next section and add plone.restapi as an add-on.
+
+Install Plone locally with plone.restapi
+----------------------------------------
+.. code-block::
+
+    git clone git@github.com:plone/plone.restapi && cd plone.restapi
+    virtualenv-2.7 . || virtualenv .
+    python bootstrap.py
```

Comment:
@tulikavijay using bootstrap is kind of deprectated. We use now:

bin/pip install -r requirements.txt (needs to exist)